### ISSUE TITLE
404 Not Found: Is modrewrite configured on your system? when trying to login

### ISSUE THREAD

**Issue by trillerpfeife** [YunoHost-Apps/emoncms_ynh#2]
the following error pops up then trying to register or login:
![image](https://user-images.githubusercontent.com/10820176/153615577-133dc722-36f7-4f5c-bdfd-8216d619f361.png)

Running: 10.8.5~ynh1
On: 4.3.6.2 (stable).

**Comment by chbmeyer** [comment]
Hello there,
I had this error, too.
Running YunoHost 11.1.21.4 (stable) now, EmonCMS is 11.3.0~ynh1.

I found an upstream bug report and this HowTo for nginx:
https://blah.cloud/miscellaneous/how-to-run-emoncms-on-nginx-and-php5-fpm/
and added `rewrite ^/(.*)$ /index.php?q=$1 last;`.

Anyway. After fixis this issue I only was able to register with a new user (SSOwat doesn't work with EmonCMS).
Now I see a buggy login-screen for EmonCMS and login does not work.
So I end with uninstalling it again, but perhaps the above link is helpful for someone different.
